cap/modules/repoimporter/models.py

```diff
@@ -139,10 +138,10 @@ def download_url(self):
                    '{self.repo.name}/legacy.tar.gz/{self.ref}'\
                 .format(self=self)
         else:
-            token = get_access_token('GITLAB')
```

Comment:
let's maybe remove this method? It's a oneliner, called only once in code.
Ideally, we would have a class method for GitAPI, that has to be overwritten for each subclass in its own way (which for gitlab would be just reading it from the config, for github oauth token from the db)